sportnet/sportnet/playgrounds/models.py

```diff
@@ -15,3 +15,15 @@ class Meta:
                                     related_name='own_playgrounds')
     players = models.ManyToManyField(get_user_model(), verbose_name='participants',
                                      related_name='in_playgrounds')
+
+
+class Sport(models.Model):
+    class Meta:
+        verbose_name = 'Kind of sport'
+        verbose_name_plural = 'Kinds of sport'
+
+    title = models.CharField(max_length=50, verbose_name='title', blank=False)
```

Comment:
Поле должно быть уникальным
```suggestion
    title = models.CharField(max_length=50, verbose_name='title', blank=False, unique=True)
```